noise/noise.py

```diff
@@ -151,8 +155,13 @@ def sendmsg(node_id, msg, plugin, request, pay=None, **kwargs):
 
 @plugin.async_method('recvmsg')
 def recvmsg(plugin, request, last_id=None, **kwargs):
-    next_id = int(last_id) + 1 if last_id is not None else len(plugin.messages)
-    if next_id <= len(plugin.messages):
+    """Receives a chat message.
+
+    Returns a `concurrent.futures.Future`.
+    Optional parameter `last_id` can be supplied to return an older message.
+    """
+    next_id = int(last_id) if last_id is not None else len(plugin.messages)
+    if next_id < len(plugin.messages):
```

Comment:
Why in the new change we are considering also the `last_id`? Why do we need this change here? I'm missing somethings